match/lib/match/nuke.rb

```diff
@@ -274,31 +286,62 @@ def delete_files!
     def certificate_type(type)
       case type.to_sym
       when :mac_installer_distribution
-        return [Spaceship.certificate.mac_installer_distribution]
+        return [
+          Spaceship::ConnectAPI::Certificate::CertificateType::MAC_INSTALLER_DISTRIBUTION
+        ]
       when :distribution
-        return [Spaceship.certificate.production, Spaceship.certificate.apple_distribution]
+        return [
+          Spaceship::ConnectAPI::Certificate::CertificateType::MAC_APP_DEVELOPMENT,
+          Spaceship::ConnectAPI::Certificate::CertificateType::IOS_DISTRIBUTION,
+          Spaceship::ConnectAPI::Certificate::CertificateType::DISTRIBUTION
+        ]
       when :development
-        return [Spaceship.certificate.development, Spaceship.certificate.apple_development]
+        return [
+          Spaceship::ConnectAPI::Certificate::CertificateType::MAC_APP_DEVELOPMENT,
+          Spaceship::ConnectAPI::Certificate::CertificateType::IOS_DEVELOPMENT,
+          Spaceship::ConnectAPI::Certificate::CertificateType::DEVELOPMENT
+        ]
       when :enterprise
-        return [Spaceship.certificate.in_house]
+        return [
+          Spaceship::ConnectAPI::Certificate::CertificateType::IOS_DISTRIBUTION
```

Comment:
Is this right? "enterprise" → `IOS_DISTRIBUTION`? 
Following the same logic seen elsewhere (e.g. `match/lib/match/importer.rb`), the `enterprise` case could be merged with the `distribution` one, maybe:

```ruby
when :distribution, :enterprise
    return [
      Spaceship::ConnectAPI::Certificate::CertificateType::MAC_APP_DEVELOPMENT,
      Spaceship::ConnectAPI::Certificate::CertificateType::IOS_DISTRIBUTION,
      Spaceship::ConnectAPI::Certificate::CertificateType::DISTRIBUTION
    ]
```

Reply:
Enterprise accounts are only allowed to make two types of distributions certs 👇 Developer ID and iOS Distribution. Developer ID can't be revoked so iOS Distribution is the only one left for us to revoke 🤷‍♂️ 

<img width="1178" alt="Screen Shot 2020-09-23 at 5 37 48 PM" src="https://user-images.githubusercontent.com/401294/94077480-8cbf0380-fdc3-11ea-8d7a-45318f22a066.png">

Reply:
Got it! Thanks for clarifying 🙏